Sources/TensorFlow/Initializers.swift

```diff
@@ -417,6 +417,30 @@ public extension Tensor where Scalar: BinaryFloatingPoint {
             seed: Tensor<Int32>([seed.0, seed.1]))
         self = standardDeviation * sample + mean
     }
+
+    /// Creates a tensor with the specified shape, randomly sampling scalars from a truncated    
+    /// normal distribution.
+    ///
+    /// Note: The generated values follow a normal distribution with specified mean and standard
+    /// deviation, except that values whose magnitude is more than 2 standard deviations from
+    /// the mean are dropped and re-picked.
+    ///
+    /// - Parameters:
+    ///   - shape: The dimensions of the tensor.
+    ///   - mean: The mean of the distribution.
+    ///   - standardDeviation: The standard deviation of the distribution.
+    ///   - seed: The seed value.
+    init(
+        truncatedNormal shape: TensorShape,
+        mean: Scalar = 0,
+        standardDeviation: Scalar = 1,
+        seed: (Int32, Int32) = Context.local.randomSeed
```

Comment:
Could you please merge with master and make this consistent with how seeds are now defined there?

Reply:
Sure thing! I'm on it.